src/devices/Ahtxx/AhtBase.cs

```diff
@@ -0,0 +1,183 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+using System.Threading;
+using UnitsNet;
+
+namespace Iot.Device.Ahtxx
+{
+    /// <summary>
+    /// AHT temperature and humidity sensor family.
+    /// Note: has been tested with AHT20 only, but should work with AHT10 and AHT15 as well.
+    /// Up to now all functions are contained in the base class, though there might be differences between the sensors types.
+    /// </summary>
+    public class AhtBase : IDisposable
+    {
+        /// <summary>
+        /// Address of AHTxx device (0x38). This address is fix and cannot be changed.
+        /// This implies that only one device can be attached to a single I2C bus at a time.
+        /// </summary>
+        public const int DeviceAddress = 0x38;
+
+        private enum StatusBit : byte // datasheet version 1.1, table 10
```

Comment:
In general, it's better to have the comment up the line especially when the line is long. This appear in multiple part of the code.

Reply:
@Ellerbach Sorry, I screwed this up. This is the AhtBase (and all the rest of the Ahtxx and Mhz19b binding) which belongs to another pull request I did weeks ago and you already reviewed and I changed. However when starting the Amg88xx binding it did start with a dedicated branch derived from my master branch with the versions as of the time being.
So all the Ahtxx and Mhz19b stuff is already done in the context of pull request #1175 .
I'm sorry for mixing this. I think I need to figure out how to revert everything except the Amgxxx stuff to the master of the iot repro.

Reply:
No problem, it happens and happened to me quite often :-)
Few tricks which helped me over time:
- I create a second remote which is based out of this dotnet/iot repo, I usually called it "dotnet". So the command would be: `git remot add dotnet https://github.com/dotnet/iot.git`
- Then you sync the main repo with your code: `git fetch dotnet`
- Then you can merge the changes for example `git merge dotnet/master`

When I create a new brach, then I can do it from the main dotnet branch like: `git checkout -b newbranchname dotnet/master`

Now for cleaning, good luck :-) Worst case if it's too mixed, just save all your new files, delete the branch, close the PR, create a new cleaned branch, copy back the code, check it in and create a new PR.

Reply:
@Ellerbach Thank you for the relief that this happens to you as well ;) All this GIT stuff is rather new to me - usually its just me and my master branch. ;)
Still I've to admit that I can't get it resolved. Meanwhile I messed my whole local repro and I had to start over again.
However, the problem is, that my master branch contains all the stuff from the other pull request which is still pending.
Could we finalize the other pull request (#1175)? If you bring it to master I could just rebase to that.
From my point of view all remarks to the other pull request are implemented and it should be good to merge.

Reply:
> Could we finalize the other pull request (#1175)?

@krwq can please have a final look? So this will help for this PR :-) Thanks!

Reply:
sure, looking now